app/dashboard/_components/RiskAndOpps/OppSection.tsx

```diff
@@ -0,0 +1,12 @@
+import Section from '@/app/dashboard/_components/RiskAndOpps/Section'
```

Comment:
i feel it's unnecessary to have both OppSection, RiskSection and Section. They all look basically the same just has different content that can be passed from the 'entire section', one should be enough